### PR TITLE
HDDS-11782. ozone debug ldb --with-keys defaults to false instead of true

### PR DESCRIPTION
**Pull request by Tejaskriya**
## What changes were proposed in this pull request?
The fallback value for --with-keys is set to true so that if the option is mentioned without a parameter, it falls back to true.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-11782 

## How was this patch tested?

Added acceptance tests in ozone-debug-ldb.robot

### CODE REVIEW

File: hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-ldb.robot [hunk 1]
```diff
@@ -55,6 +55,14 @@ Test ozone debug ldb scan
                         Should contain          ${output}       testfile1
                         Should contain          ${output}       testfile2
                         Should contain          ${output}       testfile3
+    # test key is included with --with-keys
+    ${output1} =        Execute                 ozone debug ldb --db=/data/metadata/om.db scan --cf=keyTable | jq '."\/cli-debug-volume\/cli-debug-bucket\/testfile1"'
+    ${output2} =        Execute                 ozone debug ldb --db=/data/metadata/om.db scan --cf=keyTable --with-keys | jq '."\/cli-debug-volume\/cli-debug-bucket\/testfile1"'
```
Comment (errose28) [inline hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-ldb.robot:60]: The third equivalent option to test here is `--with-keys=true`

**Review by errose28 — COMMENTED** [review]
Thanks for the fix @Tejaskriya

File: hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-ldb.robot (hunk 1 above)
Comment (Tejaskriya) [reply hadoop-ozone/dist/src/main/smoketest/debug/ozone-debug-ldb.robot:60]: Thanks for the review, I have added this case too now

**Review by errose28 — APPROVED** [review]
Thanks for fixing and adding tests @Tejaskriya

**Pull request merged**